shell/components/form/Members/MembershipEditor.vue

```diff
@@ -56,8 +56,13 @@ export default {
   },
 
   async fetch() {
+    const roleBindingRequestParams = { type: this.type, opt: { force: true } };
+
+    if (this.type === NORMAN.PROJECT_ROLE_TEMPLATE_BINDING) {
+      Object.assign(roleBindingRequestParams, { opt: { filter: { projectId: this.parentId.split('/').join(':') } } });

```

Comment:
Is there any chance of `this.parentId` might be `undefined`? If so then `this.parentId.split` would throw an error. (optional chaining)

Reply:
It shouldn't be possible for it to be undefined or null while the type is `NORMAN.PROJECT_ROLE_TEMPLATE_BINDING` since the route requires it but it's trivial to add code to ensure that that's so prior to running so I'll do that.